mcs/class/referencesource/mscorlib/system/threading/LazyInitializer.cs

```diff
@@ -156,6 +156,40 @@ public static class LazyInitializer
             return target;
         }
 
+#if MONO
```

Comment:
a few methods from coref'x LazyInitializer.
LazyInitializer can be switched to corefx implementation but will require a small change in corefx